main.py

```diff
@@ -110,20 +110,21 @@ def render():
     ang = math.atan2(
         centre[1]-mouse_pos[1], mouse_pos[0]-centre[0]
     )
-    angledisplay = font.render(f"Angle: {math.degrees(ang)}", 1, (255,255,255))
-    screen.blit(radius_display, pygame.mouse.get_pos())
+    angledisplay = font.render(f"Angle: {round(math.degrees(ang),2)}", 1, (255,255,255))
+    screen.blit(radius_display, mouse_pos)
     screen.blit(angledisplay, (mouse_pos[0], mouse_pos[1]+20))
-    pygame.display.update()
-    fps.tick(60)
 
 
-while not done:
-    counter += 1
+while True:
     for event in pygame.event.get():
-        if event.type == pygame.QUIT or pygame.key.get_pressed()[pygame.K_e]:
-            done = True
-        elif pygame.key.get_pressed()[pygame.K_s]:
-            pygame.image.save(screen, f"{counter}.jpeg")
+        if event.type == pygame.QUIT or pygame.key.get_pressed()[pygame.K_q]:
+            pygame.quit()
+            sys.exit()
+        if event.type == pygame.KEYDOWN:
+            if event.key == pygame.K_s:
+                pygame.image.save(screen, datetime.now().strftime("%Y-%m-%d-%H-%M-%S")+".png")

```

Comment:
Make the images go in a folder called `images`. Everything else looks good to me.